frontend/src/components/ConsultantRow.tsx

```diff
@@ -458,32 +457,21 @@ function DetailedBookingRows(props: {
 }
 
 async function setDetailedBookingHours(
-  bookingId: number,
   hours: number,
   bookingType: string,
   organisationName: string,
   router: AppRouterInstance,
   consultantId: string,
   engagementId: string,
   week: number,
-  setCellId: (cellId: number) => void,
 ) {
-  const url =
-    bookingId === 0
-      ? `/${organisationName}/bemanning/api/updateHours?hours=${hours}&bookingType=${bookingType}&consultantID=${consultantId}&engagementID=${engagementId}&selectedWeek=${week}`
-      : `/${organisationName}/bemanning/api/updateHours/${bookingId}?hours=${hours}&bookingType=${bookingType}`;
+  const url = `/${organisationName}/bemanning/api/updateHours?hours=${hours}&bookingType=${bookingType}&consultantID=${consultantId}&engagementID=${engagementId}&selectedWeek=${week}`;
```

Comment:
Maybe for later, but this should be a part of the body, not searchParams

Reply:
Will look at this later